src/SOS/Strike/util.cpp

```diff
@@ -2796,42 +2813,77 @@ const char *EHTypeName(EHClauseType et)
         return "UNKNOWN";
 }
 
-void DumpTieredNativeCodeAddressInfo(struct DacpTieredVersionData * pTieredVersionData,
-    const UINT cTieredVersionData, ULONG rejitID, CLRDATA_ADDRESS ilAddr, CLRDATA_ADDRESS ilNodeAddr)
+// 2.x version
+void DumpTieredNativeCodeAddressInfo_2x(struct DacpTieredVersionData_2x * pTieredVersionData, const UINT cTieredVersionData)
 {
+    ExtOut("Code Version History:\n");
+
     for(int i = cTieredVersionData - 1; i >= 0; --i)
     {
-        ExtOut("  NativeCodeVersion:  %p\n", SOS_PTR(pTieredVersionData[i].NativeCodeVersionNodePtr));
-
         const char *descriptor = NULL;
-        switch(pTieredVersionData[i].OptimizationTier)
+        switch(pTieredVersionData[i].TieredInfo)
         {
-        case DacpTieredVersionData::OptimizationTier_Unknown:
+        case DacpTieredVersionData_2x::TIERED_UNKNOWN:
         default:
             descriptor = "Unknown Tier";
             break;
-        case DacpTieredVersionData::OptimizationTier_MinOptJitted:
-            descriptor = "MinOptJitted";
-            break;
-        case DacpTieredVersionData::OptimizationTier_Optimized:
-            descriptor = "Optimized";
+        case DacpTieredVersionData_2x::NON_TIERED:
+            descriptor = "Non-Tiered";
             break;
-        case DacpTieredVersionData::OptimizationTier_QuickJitted:
-            descriptor = "QuickJitted";
+        case DacpTieredVersionData_2x::TIERED_0:
+            descriptor = "Tier 0";
             break;
-        case DacpTieredVersionData::OptimizationTier_OptimizedTier1:
-            descriptor = "OptimizedTier1";
-            break;
-        case DacpTieredVersionData::OptimizationTier_ReadyToRun:
-            descriptor = "ReadyToRun";
+        case DacpTieredVersionData_2x::TIERED_1:
+            descriptor = "Tier 1";
             break;
         }
 
-        ExtOut("    ReJIT ID:           %d\n", rejitID);
-        DMLOut("    CodeAddr:           %s  (%s)\n", DMLIP(pTieredVersionData[i].NativeCodeAddr), descriptor);
-        DMLOut("    IL Addr:            %s\n", DMLIL(ilAddr));
-        ExtOut("    ILCodeVersion:      %p\n", SOS_PTR(ilNodeAddr));
+        DMLOut("  CodeAddr:           %s  (%s)\n", DMLIP(pTieredVersionData[i].NativeCodeAddr), descriptor);
```

Comment:
Is there a reason why the output format is changing?

Reply:
I made them consistent.